tox.ini

```diff
@@ -42,4 +42,4 @@ whitelist_externals = echo
 commands =
     # Run html first so we have a report of the failing coverage results
     coverage html
-    coverage report --show-missing --fail-under=95
+    coverage report --show-missing --fail-under=93
```

Comment:
I appreciate this check, but feels like this code is as tested as I'd want. I can add a bunch more small ones, but I think this is probably more correct. I don't feel super strongly tho.

I can probably also fix it by deleting the `rtdimport` script and some others we aren't using.

Reply:
That's fine. We have some code to delete anyway. My goal with setting the minimum at 95 was to make sure we didn't slide back in coverage after getting to 95%.